src/Momento.Sdk.Incubating/SimpleCacheClient.cs

```diff
@@ -194,7 +195,7 @@ public async Task<CacheSetBatchResponse> SetBatchAsync(string cacheName, IEnumer
     /// <param name="refreshTtl">Update the dictionary TTL if the dictionary already exists.</param>
     /// <param name="ttl">TTL for the dictionary in cache. This TTL takes precedence over the TTL used when initializing a cache client. Defaults to client TTL.</param>
```

Comment:
obvs these docs will change when this pr is not a draft

Reply:
done in most recent commit